### ISSUE TITLE
Dose  infe  support objective-c  quandary 

### ISSUE THREAD

**Issue by grip3** [facebook/infer#970]
I  am analyzing  some   ios  projects，I  hope    find  some   taint flow  from  the  source code，Is  quandary    available  for   quandary ？

**Comment by jeremydubreil** [comment]
You should be able to run Quandary on Objective C code but there are very few source/sink models for iOS mobile code at the moment. If you know about common source and sinks for iOS, we can investigate adding them to the list of models.

**Comment by jvillard** [comment]
Please discuss quandary for iOS on #965, which sounds like the same question.

**Comment by grip3** [comment]
@jeremydubreil 
     Thanks  for your reply. I read question on 
https://github.com/facebook/infer/issues/965 ,my question is roughly the same as him. If you customize the rules, is it the same as Cpp's name in the .inferconfig configuration file ?  How to name the procedure property  of ios surce and sink in .inferconfig?Is it available ?

**Comment by jeremydubreil** [comment]
@grip3, yes this is an example in C++, but it should also work for Objective C: https://github.com/facebook/infer/blob/9e048c11f86a844f6b0505681e3b614132de3568/infer/tests/codetoanalyze/cpp/quandary/.inferconfig#L34

**Comment by grip3** [comment]
@jeremydubreil  
Hi,According to what you said, I have a simple class and defined the source and sink
t, but like the problem 965, I did not analyze the result. This is the class and inferconfig that I simply constructed. Is it wrong?

@interface Test : NSObject
+(int)source:(int)a;
+(void)sink:(int)param;
@end

@implementation Test
+(int)source:(int)a{

 return a;
}

+(void)sink:(int)param{
}
@end

void call() {
  int  source = [Test source:10];
  [Test sink:source];
}

The.taint config is:
"quandary-sources": [
    {
      "procedure": "Test::source",
      "kind": "Other"
    }
  ],
  "quandary-sinks": [
    {
      "procedure": "Test::sink",
      "kind": "Other",
      "index": "1"
    }
  ]
When I execute the command line"infer  -- clang -c test.m
the out is:egend:
  "F" analyzing a file
  "." analyzing a procedure
  "C" analyzer crashed
  "T" timeout: procedure analysis took too much time
  "S" timeout: procedure analysis took too many symbolic execution steps
  "R" timeout: procedure analysis took too many recursive iterations

F...
  No issues found  
Why? there is path bewween source and sink
